pennylane/ops/qubit/__init__.py

```diff
@@ -31,15 +31,59 @@
 
 from ..identity import GlobalPhase, Identity
 from ..meta import Barrier, Snapshot, WireCut
-from .arithmetic_ops import *
-from .matrix_ops import *
-from .non_parametric_ops import *
-from .observables import *
-from .parametric_ops_multi_qubit import *
-from .parametric_ops_single_qubit import *
-from .qchem_ops import *
-from .special_unitary import SpecialUnitary
-from .state_preparation import *
+from .arithmetic_ops import QubitCarry, QubitSum, IntegerComparator
+from .matrix_ops import QubitUnitary, DiagonalQubitUnitary, BlockEncode
+from .non_parametric_ops import (
+    Hadamard,
+    H,
+    PauliX,
+    X,
+    PauliY,
+    Y,
+    PauliZ,
+    Z,
+    S,
+    T,
+    SX,
+    SWAP,
+    ECR,
+    ISWAP,
+    SISWAP,
+    SQISW,
+)
+from .observables import (
+    Hermitian,
+    SparseHamiltonian,
+    Projector,
+    BasisStateProjector,
+    StateVectorProjector,
+)
+from .parametric_ops_multi_qubit import (
+    MultiRZ,
+    PauliRot,
+    PCPhase,
+    IsingXX,
+    IsingYY,
+    IsingZZ,
+    IsingXY,
+    PSWAP,
+    CPhaseShift00,
+    CPhaseShift01,
+    CPhaseShift10,
+)
+from .parametric_ops_single_qubit import RX, RY, RZ, PhaseShift, Rot, U1, U2, U3
+from .qchem_ops import (
+    SingleExcitation,
+    SingleExcitationPlus,
+    SingleExcitationMinus,
+    DoubleExcitation,
+    DoubleExcitationPlus,
+    DoubleExcitationMinus,
+    OrbitalRotation,
+    FermionicSWAP,
+)
+from .special_unitary import SpecialUnitary, TmpPauliRot
+from .state_preparation import BasisState, StatePrep, QubitDensityMatrix
 
 __ops__ = {
```

Comment:
One concern I have is with having multiple sources of truth for what is available at a certain level. Imagine that we add an operator, we would need to explicitly import this operator and at the same time add it to this `__ops__`. I don't have a good answer for this, just something to think about.